Design Patterns/Form Objects.md

```diff
@@ -78,6 +78,7 @@ We create a Form Object to represent this specific form:
 ```ruby
 class RegistrationForm
   include ActiveModel::Model
+  include ActiveModel::Validations

```

Comment:
What version of Rails are you using? In version 4 & 5, the active model validations module is already included inside of the active model model, so this is not needed.